### PR TITLE
Fix for kaytrip.com

### PR DESCRIPTION
**Pull request by felixonmars**
- Invert icons
- Invert backgrounds

### CODE REVIEW

**Comment by Gusted** [comment]
That are a lot of classes could you point out where you can find it?

**Comment by felixonmars** [comment]
They are mostly on:

kaytrip.com
bus.kaytrip.com
www.kaytrip.com/hotel.html

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2748,6 +2748,20 @@ div.output_png img
...
+INVERT
+.date
+.jd_ss
+.liuchengt
+.nApp
+.nTel
+.n_app
+.tt_t
+.wmap
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2762]: Please add

```CSS

CSS
#imageField {
    background-color: transparent !important;
}
.jd_xl {
    filter: none !important;
}
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -2748,6 +2748,28 @@ div.output_png img
...
+CSS
+#imageField {
+    background-color: transparent !important;
+}
+.jd_xl {
+    filter: none !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2768]: 4 spaces please.

Comment (felixonmars) [reply src/config/dynamic-theme-fixes.config:2768]: @Gusted I believe it's 4 spaces already ._.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:2768]: I can't count woops.

**Pull request merged**